src/containers/Schedule/index.js

```diff
@@ -36,26 +39,36 @@ export class Schedule extends Component {
       return (
         <ScheduleCard
           booking={booking}
-          type="pairer"
-          cancelPairing={this.cancelPairing}
+          person={booking.pairee}
+          key={booking.id}
         />
       );
     });
   };
 
-  cancelPairing = id => {
-    const { deletePairingThunk } = this.props;
-    deletePairingThunk(id);
-  };
-
   render() {
     const { user } = this.props;
     return (
-      <div>
-        <CurrentOpenings openings={this.getOpenings()} user={user} />
-        <div>
-          {this.showPaireeBookings()}
-          {this.showPairerBookings()}
+      <div className="Schedule">
```

Comment:
Really like the three column view, I think it cleans up how the user sees their schedule